x-pack/plugins/security_solution/public/app/search/index.test.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { getSearchDeepLinksAndKeywords } from '.';
+import { SecurityPageName } from '../../../common/constants';
+
+describe('public search functions', () => {
+  it('returns a subset of links for basic license, full set for platinum', () => {
+    const basicLicense = 'basic';
+    const platinumLicense = 'platinum';
+    for (const pageName of Object.values(SecurityPageName)) {
```

Comment:
This might been a good fit for `describe.each` or `it.each`. See https://jestjs.io/docs/en/api#describeeachtablename-fn-timeout

For now, perhaps add `expect.assertions` as described here: https://jestjs.io/docs/en/expect#expectassertionsnumber